Horizon/Horizon/Sources/Common/View/LearningObjectHeaderView.swift

```diff
@@ -19,7 +19,14 @@
 import Core
 import SwiftUI
 
-struct ContentDetailsView: View {
+struct LearningObjectHeaderView: View {
```

Comment:
I think renaming it from **LearningObjectHeaderView** to **AssignmentHeaderView** would be more meaningful.

Reply:
We'll use it at other places not just for assignments, hence the name. And Learning Object is the new term used in Horizon.